Cython/Compiler/Main.py

```diff
@@ -274,8 +277,10 @@ def c_file_out_of_date(self, source_path, output_path):
             return 1
         for kind, name in self.read_dependency_file(source_path):
             if kind == "cimport":
+                # missing suffix?
```

Comment:
No idea. As far as I can tell, this code path is only used by `Cython/Build/BuildExecutable.py`

Reply:
Not sure what you two mean here. What suffix is missing where? The `name` shouldn't have one, and the lookups don't expect one.

Reply:
ok, will remove the comments